config/dynamicconfig/development-cass.yaml

```
@@ -36,6 +36,8 @@ frontend.workerVersioningDataAPIs:
   - value: true
 frontend.workerVersioningWorkflowAPIs:
   - value: true
+system.enableDeployments:
```

do we want system.enableDeploymentVersions also true here?
```suggestion
system.enableDeploymentVersions:
  - value: true
system.enableDeployments:
```